tests/views/definitions/unnestEmptyCollectionWithNull.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Unnest empty collection with null",
+  "resource": "Patient",
+  "select": [
+    {
+      "name": "id",
+      "expr": "id"
+    },
+    {
+      "forEachOrNull": "photo",
```

Comment:
This doesn't look right? I think it would be more like this:
```
"forEachOrNull": {
  "name": "photo",
  "select": [ ... ]
}
```

Reply:
I don't think so... in the simplified JSON form `forEachOrNull` should be the FHIRPath expression to be unnested, and inside the select `name` should be the column name and `expr` the FHIRPath of the subselection.

If we were to translate it to a JSON representation of a FHIR resource based upon the logical model, I guess it would look like this:

```json
{
  "resourceType": "ViewDefinition",
  "resource": "Patient",
  "description": "Unnest empty collection with null",
  "select": [
    {
      "name": "id",
      "expression": "id",
      "forEachOrNull": {
        "expression": "photo",
        "select": [
          {
            "name": "photo_title",
            "expression": "title"
          }
        ]
      }
    }
  ]
}
```